### PR TITLE
feat(metrics): Transaction duration and mobile measurements in mock API [INGEST-680]

### PR DESCRIPTION
**Pull request by jjbayer**
As in https://github.com/getsentry/sentry/pull/30219, add more metrics and tags to the mock API.

### CODE REVIEW

File: src/sentry/snuba/metrics.py [hunk 1]
```diff
@@ -447,16 +447,40 @@ class SnubaField:
         "operations": _AVAILABLE_OPERATIONS["metrics_sets"],
         "tags": _SESSION_TAGS,
     },
+    "transaction.duration": {
```
Comment (jjbayer) [inline src/sentry/snuba/metrics.py:450]: `count(transaction.duration)` can be used to determine transactions per minute.

File: src/sentry/snuba/metrics.py [hunk 2]
```diff
@@ -447,16 +447,40 @@ class SnubaField:
...
+        "type": "distribution",
+        "operations": _AVAILABLE_OPERATIONS["metrics_distributions"],
+        "tags": {
+            **_MEASUREMENT_TAGS,
+            "transaction.status": [
+                # Subset of possible states:
+                # https://develop.sentry.dev/sdk/event-payloads/transaction/
+                "ok",
+                "cancelled",
+                "aborted",
```
Comment (jjbayer) [inline src/sentry/snuba/metrics.py:460]: `count(transaction.duration)` grouped by `transaction.status` can be used to determine failure rate.

File: src/sentry/snuba/metrics.py [hunk 3]
```diff
@@ -447,16 +447,40 @@ class SnubaField:
...
         }
-        for web_vital in ("lcp", "fcp", "fid", "cls")
+        for measurement in (
+            "lcp",
+            "fcp",
+            "fid",
+            "cls",
+            "frames_total",
+            "frames_slow",
+            "frames_frozen",
```
Comment (matejminar) [inline src/sentry/snuba/metrics.py:480]: Looks like there's also `frames_slow_rate` and `frames_frozen_rate` (slow and total divided).
The current performance product uses those two. Can we have an alias like that too?

Comment (jjbayer) [reply src/sentry/snuba/metrics.py]: I've refactored to use an existing list of metric names, which includes both: https://github.com/getsentry/sentry/blob/050e66ff0beb3c7d2b92824ca385ba55dfc48b65/src/sentry/relay/config.py#L368-L369

**Review by matejminar — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**